src/main/kotlin/io/perfometer/statistics/consumer/StatisticsTextFileWriter.kt

```diff
@@ -0,0 +1,29 @@
+package io.perfometer.statistics.consumer
+
+import io.perfometer.statistics.ScenarioSummary
+import java.io.File
+import java.nio.charset.Charset
+import java.nio.file.Files
+import java.nio.file.Paths
+
+internal object StatisticsTextFileWriter {
+
+    fun write(scenarioSummary: ScenarioSummary) {
+        val formatted: String = StatisticsTextFormatter.format(scenarioSummary)
+        val file: File = createReportFile()
+        Files.write(file.toPath(), formatted.toByteArray(Charset.forName("UTF-8")))
```

Comment:
You could use kotlin.text.Charsets.UTF_8
Or even better extension function File.writeText (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/java.io.-file/write-text.html)

Reply:
Oh, didn't know about that, thanks :-)